js/packages/web/src/views/home/setup.tsx

```diff
@@ -6,7 +6,7 @@ import {
 } from '@oyster/common';
 import { useWallet } from '@solana/wallet-adapter-react';
 import { Button } from 'antd';
-import { useCallback, useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
```

Comment:
Used?

Reply:
In general, lint demands that `React` be in scope everywhere that JSX is used, because JSX ultimately turns into…

```
React.createElement(...);
```